docs/machine-learning/resources/tasks.md

```diff
@@ -0,0 +1,46 @@
+# Machine Learning Tasks
```

Comment:
- you need to add the metadata section on top
-  sentence case for title and headings
- nit: add a new line between headings and the text